content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -282,12 +282,25 @@ must be able to receive OTLP over `grpc`. Therefore, the example uses
 
 #### Excluding auto-instrumentation {#js-excluding-auto-instrumentation}
 
-By default, the Node.js auto-instrumentation ships with
-[many instrumentation libraries](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/metapackages/auto-instrumentations-node/README.md#supported-instrumentations).
-At the moment, there is no way to opt-in to only specific packages or disable
-specific packages. If you don't want to use a package included by the default
-image you must either supply your own image that includes only the packages you
-want or use manual instrumentation.
+By default, the Node.js auto-instrumentation has all the
+[supported instrumentation libraries](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/metapackages/auto-instrumentations-node/README.md#supported-instrumentations)
+enabled.
+This makes instrumentation easy, but can result in very slow application startup time and collecting too much or unwanted data.
+
+To enable only specific instrumentations you can use the `OTEL_NODE_ENABLED_INSTRUMENTATIONS` environment variable as documented in the
+[Node.js auto-instrumentation documentation](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node#usage-auto-instrumentation).
```

Comment:
link changed with [this commit](https://github.com/open-telemetry/opentelemetry.io/pull/4137/commits/436c3bacdbd3830a71c0dbf740eda070eaef6074), that is rebased on top of latest main branch and squashed into the [commit that updated k8s operator doc](https://github.com/open-telemetry/opentelemetry.io/pull/4137/commits/beab727f5e71fdf3f68ab31e6afb67676df1ac5b)

Reply:
> This should be documentation on the website and not (only) in the repository. 

Done with [this commit](https://github.com/open-telemetry/opentelemetry.io/pull/4137/commits/bb28bfaae25269912779b74b85c7ec6447104cdc). 

> Can you provide an additional PR to update this page as well?

I'd like to avoid creating separate issue and PR just for that, unless that is absolutely needed.

Reply:
no separate PR is needed